relay-general/src/store/regexes.rs

```diff
@@ -65,7 +65,7 @@ pub static SQL_NORMALIZER_REGEX: Lazy<Regex> = Lazy::new(|| {
     # Capture `SAVEPOINT` savepoints.
     ((?-x)SAVEPOINT (?P<savepoint>(?:(?:"[^"]+")|(?:'[^']+')|(?:`[^`]+`)|(?:[a-z]\w+)))) |
     # Capture single-quoted strings, including the remaining substring if `\'` is found.
-    ((?-x)(?P<single_quoted_strs>('(?:[^']|'')*?(?:\\'.*|[^']')))) |
+    ((?-x)(?P<single_quoted_strs>'(?:\\'|[^'])*(?:'|$))) |
```

Comment:
Without this change, the test below would result in a replacement string like this:

```
'%sduration%slcp%slcp%slcp%sduration%sduration%slcp%stransaction%stransaction%s%s%s-13T09:%s:%s%sUniversal%s%s%s-11T09:%s:%s%sUniversal%sproduction'
```